modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -259,7 +259,8 @@ export default class Tile3DLayer extends CompositeLayer {
   }
 
   renderLayers() {
-    return this.state.layers;
+    const layerMapValues = Object.values(this.state.layerMap);
```

Comment:
Nit: I think we should do this when we calculate `this.state.layers`, and still return `this.state.layers` here.

Reply:
@xintongxia  planed to remove 'this.state.layers'. For now, keep the current changes.